src/robusta/integrations/jira/sender.py

```diff
@@ -219,11 +233,20 @@ def send_finding_to_jira(
             elif attr == "cluster_name":
                 labels.append(self.cluster_name)
 
-        self.client.create_issue(
+        status: FindingStatus = (
+            FindingStatus.RESOLVED if finding.title.startswith("[RESOLVED]") else FindingStatus.FIRING
+        )
+
+        # Default priority is "Major" if not a standard severity is given
+        severity = SEVERITY_JIRA_ID[finding.severity] or "Major"
```

Comment:
Here is probably also better to use `get()` to avoid `KeyError` with unknown severities